src/main/resources/application.properties

```diff
@@ -17,4 +17,10 @@ quarkus.openshift.add-name-to-label-selectors=false
 quarkus.mailer.auth-methods=DIGEST-MD5 CRAM-SHA256 CRAM-SHA1 CRAM-MD5 PLAIN LOGIN
 quarkus.mailer.host=smtp.gmail.com
 quarkus.mailer.port=465
-quarkus.mailer.ssl=true
\ No newline at end of file
+quarkus.mailer.ssl=true
+
+quarkus.native.additional-build-args=--initialize-at-run-time=org.kohsuke.github.extras.okhttp3.ObsoleteUrlFactory\\,org.kohsuke.github.extras.authorization.JWTTokenProvider\\,org.jboss.set.LotteryCommand
+quarkus.native.compression.level=1
+
+# This is publicly available access token with no right. But we would appreciate you not stealing it :)
```

Comment:
can this be somehow used for DDoS or something? Why do we need it?

Reply:
I dont think is. It's just account's access token. You can generate multiple accounts for free on redhat's jira and thus do the same wih multiple accounts. This is for the underlying library, which requires access token.